examples/bridge-app/linux/README.md

```diff
@@ -108,26 +108,26 @@ is simulated with the value/label pair `"room"`/`[light name]`.
 
 -   Install tool chain
 
-          ```
-          $ sudo apt-get install git gcc g++ python pkg-config libssl-dev libdbus-1-dev libglib2.0-dev ninja-build python3-venv python3-dev unzip
-          ```
+    ```
+    $ sudo apt-get install git gcc g++ python pkg-config libssl-dev libdbus-1-dev libglib2.0-dev ninja-build python3-venv python3-dev unzip
+    ```
 
 -   Build the example application:
 
-          ```
-          $ cd ~/connectedhomeip/examples/bridge-app/linux
-          $ git submodule update --init
-          $ source third_party/connectedhomeip/scripts/activate.sh
-          $ gn gen out/debug
-          $ ninja -C out/debug
-          ```
+    ```
+    $ cd ~/connectedhomeip/examples/bridge-app/linux
+    $ git submodule update --init
+    $ source third_party/connectedhomeip/scripts/activate.sh
+    $ gn gen out/debug
+    $ ninja -C out/debug
```

Comment:
Maybe also remove $ prefixes and add language to the markdown code block. This way one can copy the entire code block and past it directly into the terminal.

```suggestion
    ```sh
    cd ~/connectedhomeip/examples/bridge-app/linux
    git submodule update --init
    source third_party/connectedhomeip/scripts/activate.sh
    gn gen out/debug
    ninja -C out/debug
```

Reply:
Good idea! They have now been updated.